pom.xml

```diff
@@ -79,7 +79,7 @@
         <lombok.version>1.16.4</lombok.version>
         
         <springframework.version>[4.3.6.RELEASE,5.0.0.M1)</springframework.version>
-        <spring-boot.version>[1.5.0.RELEASE,2.0.0.M1)</spring-boot.version>
+        <spring-boot.version>[1.5.20.RELEASE,2.0.0.M1)</spring-boot.version>
```

Comment:
Why change the version of spring-boot

Reply:
Spring upgrade to https, the 1.5.0 version doesn't download correctly, so CI will be failure.

the dev branch upgrade the version to solve CI fail.